hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapContainer.java

```diff
@@ -106,8 +106,8 @@ public void setId(long newValue) {
         idGen = newValue + ID_PROMOTION_OFFSET;
     }
 
-    public void delete(Data dataKey) {
-        multiMapValues.remove(dataKey);
+    public boolean delete(Data dataKey) {
```

Comment:
We need to know if a delete was successful, to decide if `lastUpdateTime` should be modified